library/kotlin/test/io/envoyproxy/envoymobile/EnvoyClientBuilderTest.kt

```diff
@@ -29,6 +31,16 @@ class EnvoyBuilderTest {
     assertThat(envoy.logLevel).isEqualTo(LogLevel.DEBUG)
   }
 
+  @Test
+  fun `specifying stats domain overrides default`() {
+    clientBuilder = EnvoyClientBuilder(Domain("api.foo.com"))
+    clientBuilder.addEngineType { engine }
+
+    clientBuilder.addStatsDomain("stats.foo.com")
+    val envoy = clientBuilder.build()
+    assertThat(envoy.envoyConfiguration!!.statsDomain).isEqualTo("stats.foo.com")
```

Comment:
@rebello95 I noticed that we don't have a mirror of these tests (ones that test clientBuilder.add<Config>()` in swift, or did I miss them?

Reply:
We have similar tests in `EnvoyClientBuilderTests` but it looks like we don't have specific cases for adding individual properties to the builder. Would you mind adding some to that file?

Reply:
done.